engine/components-api/src/com/cloud/network/IpAddressManager.java

```diff
@@ -43,6 +43,9 @@
         "If true, when account has dedicated public ip range(s), once the ips dedicated to the account have been consumed ips will be acquired from the system pool",
         true, ConfigKey.Scope.Account);
 
+    static final ConfigKey<Boolean> RulesContinueOnError = new ConfigKey<Boolean>("Advanced", Boolean.class, "network.rule.delete.ignoreerror", "true",
```

Comment:
The default for ignoreerror is set to true, means the errors will be ignored.
Shouldn't the default be set to false ?